scripts/ci/allFilesChangedInPatterns.mjs

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+import {setOutput} from '@actions/core';
+import {minimatch} from 'minimatch';
+import {getBaseHeadSHAs, getChangedFiles} from './git-utils.mjs';
+
+function getOutputName() {
+  return process.argv[2];
+}
+
+function getPatterns() {
+  return process.argv.slice(3);
+}
+
+function everyFileMatchOnePattern(files, patterns) {
+  return files
+    .trim()
+    .split(/\r?\n/)
+    .every((file) => {
+      return patterns.some((pattern) => {
+        return minimatch(file.trim(), pattern);
+      });
+    });
+}
+
+const {base, head} = getBaseHeadSHAs();
+const files = getChangedFiles(base, head);
+const patterns = getPatterns();
```

Comment:
Nitpick, but I don't see any added value in having `getPatterns` and ` getOutputName` functions; we could just do:

```typescript
const patterns = process.argv.slice(3);
// ...
const outputName = process.argv[2];
```

and it'd be as readable IMO.

Same comment in scripts/ci/someFilesChangedInPatterns.mjs

Reply:
I copied those from the already existing script that I didn't do :D 
I didn't change those as much as possible so it kept the same structure as the other CI script file.

So I would prefer to keep those like that to match the other file of `someFilesChangedInPatterns` :)